charts/testkube-api/values.yaml

```diff
@@ -52,6 +52,23 @@ image:
   digest: ""
   pullSecret: []
 
+imageTwToolkit:
+  registry: docker.io
+  repository: kubeshop/testkube-tw-toolkit
+  ## Overrides the image tag whose default is the api-server version.
+  # tag: "1.7.24"
+  digest: ""
+  pullSecret: []
+
+
+imageTwInit:
+  registry: docker.io
+  repository: kubeshop/testkube-tw-init
+  ## Overrides the image tag whose default is the api-server version.
+  # tag: "1.7.24"
+  digest: ""
+  pullSecret: []
```

Comment:
`pullSecrets` (plural), also based on @dejanzele's comment, probably worth to keep to pull policy. We don't support either of them yet, but we could add support in the future, if someone will actually need it.

Alternatively, we can delete both for now, and add it later, when it will be supported. WDYT @dejanzele?

Reply:
done, removed

Reply:
yep, let's make it work in this PR and after that we can improve